docs/src/docs/asciidoc/user/xml.adoc

```diff
@@ -16,15 +16,14 @@ Using an XML file you can configure a `CacheManager` at creation time, according
 
 === `<config>` root element
 
-The root element of our XML configuration. One `<config>` element and, by implication, one XML file,
-provides the definition for a `CacheManager`.  With Ehcache 3, however, you may create multiple
-`CacheManager` instances using the same XML configuration file.  Unlike the JSR-107
-`javax.cache.spi.CachingProvider`, Ehcache does not maintain a registry of `CacheManager` instances.
+The root element of our XML configuration. One `<config>` element in an XML file provides the definition for a `CacheManager`.  
 
+Note: Ehcache allows for creating multiple `CacheManager` instances using the same XML configuration file. 
```

Comment:
I would instead use NOTE in all caps, which generates a note box, as used further down in this document.